lib/tool_belt/cherry_picker.rb

```diff
@@ -92,7 +93,11 @@ def write_cherry_pick_log(picks, release)
     end
 
     def log_entry(pick)
-      "#{pick['id']} - #{Time.parse(pick['closed_on'])}: [#{pick['revision']}] #{pick['subject']}"
+      if pick['bugzilla_id']
+        "#{pick['bugzilla_id']} - #{pick['id']} : [#{pick['revision']}] #{pick['subject']}"
+      else
+        "#{pick['id']} - #{Time.parse(pick['closed_on'])}: [#{pick['revision']}] #{pick['subject']}"
+      end
```

Comment:
This will change the output based upon if a Redmine is hooked to a bugzilla which would create uneven and varying output. I think it may be better to just add in the bugzilla ID within the entry if its present. This is where the use of ERB templating would come in way handier than my ad-hoc method.